src/mocks/assets.ts

```diff
@@ -1,7 +1,7 @@
+import { Asset } from '@cognite/sdk/dist/src/types/types';
```

Comment:
Everywhere you do this (import from `@cognite/sdk/dist/src/types/types`) could you replace it with `import { INTERFACE_TO_IMPORT } from '@cognite/client`; Example:
`import { Asset } from '@cognite/sdk';`

Reply:
fixed all!